src/test/java/cmput402/tdd/AppTest.java

```diff
@@ -265,4 +265,269 @@ public void testDisplayPeople() {
         assertEquals(app.displayPeople(), out);
         
     }
+  
+    @Test
+    public void testEditRecipe(){
+        App app = new App();
+        Recipe recipe = new Recipe("apple pie");
```

Comment:
Still need to use mocking here instead of instantiation
```suggestion
        Recipe recipe = mock(Recipe.class);
        when(recipe.getName()).thenReturn("apple pie");
```

Reply:
Thanks for the suggested code, made requested change